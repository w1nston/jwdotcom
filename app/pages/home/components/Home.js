import React from 'react';

export default function Home() {
  return (
    <section className="main-content-container__home">
      This is not optimal, but life is not about optimal, it's about learning.
    </section>
  );
}
