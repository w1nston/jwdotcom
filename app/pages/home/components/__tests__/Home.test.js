import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Home from '../Home';

test('that Home component is rendered', () => {
  const component = shallow(<Home />);
  expect(shallowToJson(component)).toMatchSnapshot();
});
