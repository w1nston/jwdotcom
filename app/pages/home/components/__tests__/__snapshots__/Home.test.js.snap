exports[`test that Home component is rendered 1`] = `
<section
  className="main-content-container__home">
  This is not optimal, but life is not about optimal, it\'s about learning.
</section>
`;
